pkg/cmd/pulumi/pulumi.go

```diff
@@ -180,6 +180,8 @@ func NewPulumiCmd() *cobra.Command {
 		"Enable verbose logging (e.g., v=3); anything >3 is very verbose")
 	cmd.PersistentFlags().StringVar(
 		&color, "color", "auto", "Colorize output. Choices are: always, never, raw, auto")
+	cmd.PersistentFlags().BoolVar(&httpstate.PrintFullStackNames, "full-name", false,
```

Comment:
I don't think we'll want to couple this flag directly into global state in the `httpstate` module.  Some more notes in summary, but would love to find a way to scope this a bit more if possible.